spec/requests/admins_spec.rb

```diff
@@ -14,18 +14,31 @@
 
     before do
       create(:collection_version_with_collection, collection: work_collection)
+      create(:embargo_lifted_event, eventable: work1, created_at: 3.days.ago)
```

Comment:
we might want these lines only in the scope that is for items_recent_activity or they get created even on requests where they're not used.